opm/autodiff/BlackoilWellModel.hpp

```diff
@@ -61,9 +61,14 @@
 
 namespace Opm {
 
+        template<typename TypeTag>
+        class BlackoilModelEbos;
+
         /// Class for handling the blackoil well model.
         template<typename TypeTag>
         class BlackoilWellModel {
+            // Needs acces to wells_ecl_
+            friend class BlackoilModelEbos<TypeTag>;
```

Comment:
Sorry for my previous message. I did not see where `wells_ecl_` is actually used there.  Or can we just introduce a public function for `wells_ecl_`?

Reply:
No need to sorry. It turns out that is not needed anymore after the rebase to accomodate the refactored well code. Nice catch.